1-python-question.py

```diff
@@ -25,8 +25,51 @@
 
 
 def processRequest(request):
-    # Your code here
-    return
+
+    class EmptyApplicantsException(Exception):
+        pass
+
+    try:
+        applicants = request["applicants"]
+        if len(applicants) == 0:
+            raise EmptyApplicantsException
+    except EmptyApplicantsException:
+        return {"error": "No applicants"}
+
+    to_return = {}
+
+    successfulApplicants = list(filter(lambda x: x if x not in bannedVisitors else None, applicants ))
+    bannedApplicants = list(filter(lambda x: x if x in bannedVisitors else None, applicants ))
```

Comment:
The 2 lambda expressions can be simplified to 
`lambda x: x not in bannedVisitors`
`lambda x: x in bannedVisitors`